v3io/dataplane/request.py

```diff
@@ -104,7 +104,8 @@ def encode_get_object(container_name, access_key, kwargs):
     offset = kwargs.get("offset")
 
     # if the append flag is passed, add a range header
```

Comment:
Instead of a new comment, please update this wrong comment:
`if the offset param is specified, add a range header`